### PR TITLE
Milestone 1: Create book app list with plain object.

### PR DESCRIPTION
**Pull request by MateeSaafi**
### **In this milestone, I achieved the following...**

- Set up the branch
- Create a form
- Create a list dynamically
- Get the user's input
- Save user's input into local storage
- Send the user's input to the user interface
- Delete items both from the user's interface and local storage

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @matee-safi and @Cabraham1,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### All the links on the Table of Contents should correctly lead to a section in the readme file. Some of your links lead nowhere. Kindly add the sections as they were in the template and link them properly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const form = document.getElementById('form');
+// const list = document.querySelector('.table-part');
```
Comment (julie-ify) [inline script.js]: - [ ] Kindly remove commented code.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+books.forEach((book) => {
+  displayBooks(book.id, book.title, book.author);
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const title = document.getElementById('title');
+    const author = document.getElementById('author');
+    // if (title.value && author.value) {
```
Comment (julie-ify) [inline script.js]: - [ ] Kindly remove commented code.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @matee-safi and @Cabraham1,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (Cabraham1) [reply script.js]: all code comments has been remove from my code base thank you

Comment (Cabraham1) [reply script.js]: all code comments has been remove from my code base thank you

File: script.js (hunk 1 above)
Comment (Cabraham1) [reply script.js]: all code comments has been remove from my code base thank you

**Review by Whoistolu — APPROVED** [review]
### Hi Team,
@matee-safi and @Cabraham1 

Great job implementing the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Cabraham1** [comment]
I'm grateful that you reviewed and approved my project. @Whoistolu

**Pull request merged**